common/djangoapps/student/tests/test_models.py

```diff
@@ -728,3 +730,53 @@ def _set_up_invited_student(self, course, active=False, enrolled=True, course_mo
                 course_enrollment.save()
 
         return user
+
+
+@skip_unless_lms
+class EnrollmentEventsTest(SharedModuleStoreTestCase):
```

Comment:
I'm not sure whether this is the best place to implement tests for events, should we move them to their own file? @felipemontoya

Reply:
Maybe common/djangoapps/student/tests/test_events.py is a better place?